docs/modules/release-notes/pages/5-4-0.adoc

```diff
@@ -0,0 +1,353 @@
+* Removed the beta annotations from the MongoDB classes.
+https://github.com/hazelcast/hazelcast/pull/25743[#25743]
+* Added TLS support for MongoDB data connections.
+https://github.com/hazelcast/hazelcast/pull/25301[#25301]
+* Added Oracle database support to the JDBC SQL connector.
+https://github.com/hazelcast/hazelcast/pull/25255[#25255]
+* Added support for inline Avro schemas for Kafka mappings.
+https://github.com/hazelcast/hazelcast/pull/25207[#25207]
+* Added support for `DataSerializable` serialization to Mongo connector.
+https://github.com/hazelcast/hazelcast/pull/25197[#25197]
+* Check for existing resources for Mongo connector is now done only once; previously, it was performed on every processor.
+https://github.com/hazelcast/hazelcast/pull/24953[#24953]
+* Hazelcast JDBC connector now supports Microsoft SQL server.
+https://github.com/hazelcast/hazelcast/pull/25071[#25071]
+* Added the ability to configure the pool size of a MongoDB data connection. See xref:sql:mapping-to-mongo.adoc#creating-a-mongodb-mapping[Creating a MongoDB Mapping].
+https://github.com/hazelcast/hazelcast/pull/25027[#25027]
+
+=== Data Structures
+
+* Added support of specifying the maximum size of a CPMap.
+https://github.com/hazelcast/hazelcast/pull/25881[#25881]
+* Added Java client support for CPMap.
+https://github.com/hazelcast/hazelcast/pull/25877[#25877]
+* Added check for negative positions on the collections' `getItemAtPositionOrNull()` method.
+https://github.com/hazelcast/hazelcast/pull/25672[#25672]
+* Introduced a cluster state check to improve the removal of expired map/cache entries from the cluster.
+The removal operation is no longer executed if the cluster is in passive state.
+https://github.com/hazelcast/hazelcast/pull/24808[#24808]
+* Added the `IMap.localValues()` and `IMap.localValues(Predicate)` methods for a faster access to locally owned values in maps.
+https://github.com/hazelcast/hazelcast/pull/24673[#24763]
+* Added the `deleteAsync()` for maps to asynchronously remove a provided map entry key.
+https://github.com/hazelcast/hazelcast/pull/23509[#23509]
+* Fixed the Javadoc for caches where it was referring incorrectly to statistics instead of management, for the `setManagementEnabled()` method.
+https://github.com/hazelcast/hazelcast/pull/22575[#22575]
+* Implemented RAFT lifecycle interfaces for CPMap. #6800
+* Added the `getCPObjectInfos()` method to create a snapshot of all existing CP objects for a given service on the group leader. #615
+* Added the `getCPGroupIds()` method to the CP Subsystem API to allow listing all data structures within a CP group. #591
+* The "last update time" record of the map entries recovered from the disk persistence is not used anymore when calculating the entries' expiration times. #233
+* [.enterprise]*Enterprise* Added the ability to check whether the `ADVANCED_CP` feature is present in the Hazelcast Platform Enterprise license, to enable CPMap. #157
```

Comment:
just a license check PR, not RN-worthy I think. The requirements for using CPMap are anyway listed in the CPMap documentation page.